.env

```diff
@@ -15,7 +15,7 @@ LOCAL_PORT=8889
 LOCAL_DIR=src
 
 # The PHP version to use. Valid options are 'latest', and '{version}-fpm'.
-LOCAL_PHP=latest
+LOCAL_PHP=8.1-fpm
```

Comment:
Do we need these changes?

Reply:
Thanks for flagging this @mukeshpanchal27. Nope, this should not be changed.

Reply:
It's now been rolled back to its original state. Thanks again :)